adafruit_wiznet5k/adafruit_wiznet5k_socket.py

```diff
@@ -216,18 +217,19 @@ def accept(self):
         return client_sock, addr
 
     def connect(self, address, conntype=None):
-        """Connect to a remote socket at address. (The format of address depends
-        on the address family — see above.)
+        """Connect to a remote socket at address.
         :param tuple address: Remote socket as a (host, port) tuple.
-
         """
         assert (
             conntype != 0x03
         ), "Error: SSL/TLS is not currently supported by CircuitPython."
         host, port = address
 
         if hasattr(host, "split"):
-            host = tuple(map(int, host.split(".")))
+            try:
```

Comment:
This is great - thanks for adding this!

Could we also split out the `proto` and `raise` for if a user passes a `https:` URL?
https://github.com/adafruit/Adafruit_CircuitPython_Requests/blob/master/adafruit_requests.py#L532

Reply:
Uhm, this is a socket we're talking about here.  Shouldn't have either "http:" or "https:".